_posts/2024-06-18-trino-fest-recap.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "Trino Fest 2024 recap"
+author: "Manfred Moser, Cole Bowden, Monica Miller"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2024/recap-banner.png
+---
+
+tbd
+Trino Fest 2024 is successfully in the books! While over 100 enthusiastic
+members of the community gathered in Boston, over 650 virtual attendees
+joined us worldwide to learn from our expert speakers as they discussed
+topics such as table formats, enhancements and optimizations, and use
+cases with Trino both large and small.
+<!--more-->
+
+## Re
+tbd
+
+
+## Sessions
+
+* [What’s new in Trino this summer](https://www.youtube.com/watch?v=mk3n0_tAdZY){:target="_blank"}
+  presented by Martin Traverso from
+  [Starburst](https://www.starburst.io){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Reducing query cost and query runtimes of Trino powered analytics platforms](https://www.youtube.com/watch?v=11yhPXIXiBY){:target="_blank"}
+  presented by Jonas Irgens Kylling from
+  [Dune](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Enhancing Trino's query performance and data management with Hudi: innovations and future](https://www.youtube.com/watch?v=JMzS2BbeK0E){:target="_blank"}
+  presented by Ethan Guo from
+  [Onehouse](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Trino Engineering @ Microsoft](https://www.youtube.com/watch?v=t7ndqYUhKSA){:target="_blank"}
+  presented by George Fisher and Ishan Patwa from
+  [Microsoft](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Enhancing data governance in Trino with the OpenLineage integration](https://www.youtube.com/watch?v=A7hj1M7IYj8){:target="_blank"}
+  presented by Alok Kumar Prusty from
+  [Apple](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Best practices and insights when migrating to Apache Iceberg for data engineers](https://www.youtube.com/watch?v=dKQ2zShNlyQ){:target="_blank"}
+  presented by Amit Gilad from
+  [Cloudinary](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Trino query intelligence: insights, recommendations, and predictions](https://www.youtube.com/watch?v=K3iSXOJNaSQ){:target="_blank"}
+  presented by Marton Bod from
+  [Apple](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [The open source journey of the Trino Delta Lake Connector](https://www.youtube.com/watch?v=mPfRYdvDcMo){:target="_blank"}
```

Comment:
Marius went into a deep dive on all the work and collaboration that's gone into making the Delta Lake connector in Trino a robust, first-class connector. Casual discussions, engineers working together, GitHub issues filed by the community, and innovative contributions have all come together, and Marius' talk shows why an open source community is so powerful.